geetools/ImageCollection.py

```diff
@@ -1740,3 +1740,102 @@ def to_int(d):
         ax = plot_data("doy", data, "Day of Year", colors, ax)
 
         return ax
+
+    def reduceRegion(
+        self,
+        reducer: str,
+        idProperty: str = "system:index",
+        idPropertyType: type = ee.Number,
+        geometry: ee.Geometry | None = None,
+        scale: int | float | None = None,
+        crs: str | None = None,
+        crsTransform: list | None = None,
+        bestEffort: bool = False,
+        maxPixels: int | None = None,
+        tileScale: int = 1,
+        **kwargs,
```

Comment:
``` suggestion 
        dateFormat: str = EE_DATE_FORMAT,
        numberFormat: str = "%s",
        **kwargs,
```
in `dateByBands` I didn't propose this 'cos the final goal was charting and I expect the charting prosses to handle this, but in this case, which is for general purposes, I think the user should be able to choose the date formatting.

Reply:
solved in https://github.com/gee-community/geetools/pull/359/commits/7fd2e53152d56f04c11fd13dd17516e7067a626f